Packages/com.unity.inputsystem/InputSystem/Plugins/XR/Vive.cs

```diff
@@ -89,4 +89,23 @@ protected override void FinishSetup(InputDeviceBuilder builder)
             deviceAngularVelocity = builder.GetControl<Vector3Control>("deviceAngularVelocity");
         }
     }
+
+    [InputControlLayout]
```

Comment:
Technically you don't even need these. When registering a type as a layout, it takes the type whether the attribute is there or not. The attribute just allows controlling some settings on the layout.

Reply:
I don't mind them being there, though. Kinda makes it a little more explicit.